src/vdaf/poplar1.rs

```diff
@@ -2023,6 +2050,17 @@ mod tests {
         }
     }
 
+    #[test]
+    fn agg_param_prefix_with_trailing_bits() {
+        let encoded = [
+            0, 11, // level
+            0, 0, 0, 1, // number of prefixes
+            1, 7, // encoded prefix with trailing bits
+        ];
+        let err = Poplar1AggregationParam::get_decoded(&encoded).unwrap_err();
+        assert_matches!(err, CodecError::UnexpectedValue);
+    }
+
     #[test]
```

Comment:
@divergentdave this test passes, but I'm not sure if it's for the reason we expect. Do you recall what "wrong unused bit" means?

Reply:
This comes from 6f253b3b90f3448f6cbd09ed6c1f254e4e9409e2, so it's similar to the new test above, confirming that unused bits of encoded prefixes are cleared.

Reply:
Ok, that's what I thought ... Just wanted to make sure. I've merged the two test cases into one test.